inc/cxplat.hpp

```diff
@@ -19,46 +19,33 @@
 #include "cxplat.h"
 #include "cxplat_sal_stub.h"
 
-typedef void* CxPlatCallback(
-    _Inout_ void* Context
-);
-
-class CxPlatAsync {
+template <typename T>
+class CxPlatAsyncT {
```

Comment:
Wait, I don't get it. class name is CxPlatAsyncT but the test can still use CxPlatAsync and passes?

Reply:
There's a typedef at the end of the hpp.

Reply:
Oh there's a typedef. I get it now.